crates/solidity/testing/snapshots/bindings_output/modifiers/with_args/generated/0.5.0-success.txt

```diff
@@ -0,0 +1,44 @@
+# This file is generated automatically by infrastructure scripts. Please don't edit by hand.
+
+References and definitions: 
+    ╭─[input.sol:1:1]
+    │
+  1 │ contract FunctionModifier {
+    │          ────────┬───────  
+    │                  ╰───────── def: 1
+  2 │     address public owner;
+    │                    ──┬──  
+    │                      ╰──── def: 2
+    │ 
+  4 │     modifier validAddress(address _addr) {
+    │              ──────┬─────         ──┬──  
+    │                    ╰───────────────────── def: 3
+    │                                     │    
+    │                                     ╰──── def: 4
+  5 │         require(_addr != address(0), "Not valid address");
```

Comment:
I suggest removing the `require()` from both of these tests, since they are testing modifiers, and `require()` is just adding noise for the earlier versions ..
We can add a separate test for it to still cover it..

Reply:
Good point. I think I have the same issue in several tests.

Reply:
On second thought, these _are_ testing something relevant: that the body of the modifier can access the arguments passed in (or the state variables in the `simple` case). I replaced `require()` by `assert()` which exists in all supported versions.